ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -9,18 +8,11 @@
 from posthog.models.group.util import create_group
 from posthog.models.group_type_mapping import GroupTypeMapping
 from posthog.queries.funnels.funnel import ClickhouseFunnel
-from posthog.queries.funnels.funnel_persons import ClickhouseFunnelActors
-from posthog.queries.funnels.test.breakdown_cases import funnel_breakdown_test_factory
-from posthog.queries.funnels.test.test_funnel import _create_action, funnel_test_factory
-from posthog.test.base import ClickhouseTestMixin, _create_event, _create_person
+from posthog.test.base import APIBaseTest, ClickhouseTestMixin, _create_person
 from posthog.test.test_journeys import journeys_for
 
 
-class TestFunnelBreakdown(ClickhouseTestMixin, funnel_breakdown_test_factory(ClickhouseFunnel, ClickhouseFunnelActors, _create_event, _create_action, _create_person), funnel_breakdown_group_test_factory(ClickhouseFunnel, ClickhouseFunnelActors, _create_event, _create_action, _create_person)):  # type: ignore
```

Comment:
Specific to breakdown factory tests, these are intended to be run multiple times -> What's changing is the type of funnel that's running these tests (some for ordered funnel, some for unordered funnel, etc. etc.)

Reply:
Ah, I see, the first factory is duplicated in posthog & ee, while the second one isn't, so just

```
class TestFunnelBreakdown(ClickhouseTestMixin, funnel_breakdown_group_test_factory(ClickhouseFunnel, ClickhouseFunnelActors, _create_event, _create_action, _create_person)): # type: ignore
```

is good here!

Reply:
Actually surely we should run those against Strict and Unordered too? have added that now.